Source/WebCore/Modules/webauthn/fido/AuthenticatorGetInfoResponse.cpp

```diff
@@ -115,7 +115,7 @@ Vector<uint8_t> encodeAsCBOR(const AuthenticatorGetInfoResponse& response)
 
     CBORValue::ArrayValue versionArray;
     for (const auto& version : response.versions())
-        versionArray.append(version == ProtocolVersion::kCtap ? kCtap2Version : kU2fVersion);
+        versionArray.append(version == ProtocolVersion::kCtap ? kCtap2Version.characters() : kU2fVersion);
```

Comment:
Not sure why this has to be changed given there’s a new overload for `ASCIILiteral` added.

Reply:
Probably not needed, will check and update.